### PR TITLE
Approve any namespace pending modules

### PR DESCRIPTION
**Pull request by Buckram123**
### Checklist

- [x] CI is green.
- [x] Changelog updated.

### CODE REVIEW

File: framework/packages/abstract-interface/src/native/version_control.rs [hunk 1]
```diff
@@ -139,22 +139,30 @@ impl<Chain: CwEnv> VersionControl<Chain> {
 
     /// Approve any abstract-namespaced pending modules.
     pub fn approve_any_abstract_modules(&self) -> Result<(), crate::AbstractInterfaceError> {
-        let proposed_abstract_modules = self.module_list(
+        self.approve_any_namespace_modules(Namespace::unchecked(ABSTRACT_NAMESPACE))
+    }
+
+    /// Approve any "namespace" pending modules.
+    pub fn approve_any_namespace_modules(
```
Comment (CyberHoward) [inline framework/packages/abstract-interface/src/native/version_control.rs:146]: Would like to rename this as this suggests we approve modules for any namespace. 
```suggestion
    pub fn approve_any_module_for_namespace(
```

File: integrations/Cargo.toml [hunk 2]
```diff
@@ -61,7 +61,7 @@ abstract-std = { version = "0.22.1" }
 
 abstract-adapter-utils = { version = "0.22.1" }
 abstract-dex-standard = { version = "0.22.1" }
-abstract-money-market-standard = { version = "0.22.1" }
```
Comment (CyberHoward) [inline integrations/Cargo.toml:64]: curious why this bumped

Comment (Buckram123) [reply integrations/Cargo.toml:64]: > curious why this bumped

Nicolas published a fix for standard

File: framework/packages/abstract-interface/src/native/version_control.rs (hunk 1 above)
Comment (Buckram123) [reply framework/packages/abstract-interface/src/native/version_control.rs]: updated!

**Review by CyberHoward — APPROVED** [review]

**Pull request merged**